### DISCUSSION TITLE
Compare two run results in Artillery Cloud

### DISCUSSION THREAD

**Discussion by vikramjb** [artilleryio/artillery#2705] [Artillery]
Hello @hassy @bernardobridge @InesNi 👋 

Is it possible to compare two test runs on Artillery Cloud? This will help us determine if the performance has increased or decreased between release or code changes.

**Comment by bbeesley** [comment]
I'm also interested in this. Something like the [baseline compare](https://grafana.com/docs/grafana-cloud/testing/k6/analyze-results/test-comparison/) in k6 would be ideal.
